index.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+Strip leading whitespace from each line in a string.
+
+The line with the least number of leading whitespace, ignoring empty lines, determines the number to remove.
+
+@example
+```
+import stripIndent = require('strip-indent');
+
+const string = '\tunicorn\n\t\tcake';
+//	unicorn
```

Comment:
Can you open an issue on VS Code about supporting either `/* */` comments unescaped inside codeblocks in doc comments (preferred) or alternatively/additionally support escaped `\/* */` comments inside doc comments (meaning it's not displayed as escaped). It's silly that we have to use line comments like this. It makes it harder to read the example output.

Reply:
Opened an [issue](https://github.com/Microsoft/vscode/issues/72674).